### DISCUSSION TITLE
chat-meilisearch Usage

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#240] [Q&A]
Does it mean that Langchain can be used if text files and PDF files are placed in the meili_data directory?

chat-meilisearch |
chat-meilisearch | 888b     d888          d8b 888 d8b                                            888
chat-meilisearch | 8888b   d8888          Y8P 888 Y8P                                            888
chat-meilisearch | 88888b.d88888              888                                                888
chat-meilisearch | 888Y88888P888  .d88b.  888 888 888 .d8888b   .d88b.   8888b.  888d888 .d8888b 88888b.
chat-meilisearch | 888 Y888P 888 d8P  Y8b 888 888 888 88K      d8P  Y8b     "88b 888P"  d88P"    888 "88b
chat-meilisearch | 888  Y8P  888 88888888 888 888 888 "Y8888b. 88888888 .d888888 888    888      888  888
chat-meilisearch | 888   "   888 Y8b.     888 888 888      X88 Y8b.     888  888 888    Y88b.    888  888
chat-meilisearch | 888       888  "Y8888  888 888 888  88888P'  "Y8888  "Y888888 888     "Y8888P 888  888
chat-meilisearch |
chat-meilisearch | Config file path:    "none"
chat-meilisearch | Database path:               "./data.ms"
chat-meilisearch | Server listening on: "http://meilisearch:7700"
chat-meilisearch | Environment:         "development"
chat-meilisearch | Commit SHA:          "572608c121d960cdf48c2c289a6dfd3dc834873a"
chat-meilisearch | Commit date:         "2023-02-23T13:52:56+00:00"
chat-meilisearch | Package version:     "1.0.2"
chat-meilisearch |
chat-meilisearch | Thank you for using Meilisearch!
chat-meilisearch |
chat-meilisearch |
chat-meilisearch | We collect anonymized analytics to improve our product and your experience. To learn more, including how to turn off analytics, visit our dedicated documentation page: https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html
chat-meilisearch |
chat-meilisearch | Anonymous telemetry: "Enabled"
chat-meilisearch | Instance UID:                "a9e7d0bf-d643-4906-9156-42c7d33513e5"
chat-meilisearch |
chat-meilisearch | A master key has been set. Requests to Meilisearch won't be authorized unless you provide an authentication key.
chat-meilisearch |
chat-meilisearch | Documentation:               https://docs.meilisearch.com
chat-meilisearch | Source code:         https://github.com/meilisearch/meilisearch
chat-meilisearch | Contact:             https://docs.meilisearch.com/resources/contact.html
chat-meilisearch |
chat-meilisearch | [2023-05-11T06:24:31Z INFO  actix_server::builder] Starting 5 workers
chat-meilisearch | [2023-05-11T06:24:31Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime

**Comment by danny-avila** [comment]
No it is used to search for messages only. I am looking to integrate document parsing, but it may not be with the first release of plugins as it will be more involved with UI changes
